gammapy/estimators/map/tests/test_ts.py

```diff
@@ -412,6 +413,14 @@ def test_ts_map_stat_scan(fake_dataset):
     assert_allclose(combined_map.ts.data, 2 * ts, rtol=1e-4)
     assert_allclose(combined_map.norm.data[success], norm[success], rtol=5e-2)
 
+    combined_results = get_combined_flux_maps(
+        estimator, [dataset, dataset.copy()], method="distrib"
+    )
+    combined_map = combined_results["flux_maps"]
+    assert len(combined_results["estimator_results"].values()) == 2
+    assert_allclose(combined_map.ts.data, 2 * ts, rtol=1e-4)
+    assert_allclose(combined_map.norm.data[success], norm[success], rtol=5e-2)
+
```

Comment:
Do you test that all keys mentioned into the docstring are present?

Reply:
yes it in the existing tests for `combine_flux_maps` as `get_combined_flux_maps` is build on top of it it don't retest everything